.github/workflows/release_next.yml

```diff
@@ -110,18 +110,45 @@ jobs:
         with:
           name: coverage-${{ matrix.os }}-${{ matrix.nodeRun }}
           path: coverage.tar
-  test_docs:
+  build_docs:
     if: github.repository_owner == 'pnp'
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-node@v3
         with:
-          python-version: 3.x
-      - run: pip install -r docs/pip_requirements.txt
-      - run: mkdocs build --strict
+          node-version: 18
+
+      - name: Install dependencies
+        run: npm ci
         working-directory: docs
+        
+      - name: Build docs
+        run: npm run build
+        working-directory: docs
+        
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v1
+        with:
+          path: docs/build
+  deploy_docs:
```

Comment:
We should publish docs only after we published the package to npm. Otherwise, it could happen that we update docs pointing to a release that isn't out there. Let's keep the original order.